flink-connectors/flink-sql-connector-elasticsearch7/pom.xml

```diff
@@ -142,6 +144,10 @@ under the License.
 									<pattern>com.github.mustachejava</pattern>
 									<shadedPattern>org.apache.flink.elasticsearch7.shaded.com.github.mustachejava</shadedPattern>
 								</relocation>
+								<relocation>
+									<pattern>net.jpountz</pattern>
+									<shadedPattern>org.apache.flink.elasticsearch7.shaded.net.jpountz</shadedPattern>
+								</relocation>
```

Comment:
Can you explain this relocation?

Reply:
Bumping elastic brought in this dependency, to keep the jar consistent I relocated it like the other ones